packages/code-du-travail-frontend/src/modules/enterprise/EnterpriseAgreementSearch/EnterpriseAgreementSearchInput.tsx

```diff
@@ -0,0 +1,318 @@
+              },
+            }),
+          }}
+        />
+        <LocationSearchInput
+          onLocationChange={setLocation}
+          defaultValue={location}
+          className={fr.cx(
+            "fr-col-12",
+            "fr-col-xl",
+            "fr-col-md",
+            getStateMargin(),
+            "fr-mt-2w",
+            "fr-mt-md-0"
+          )}
+        />
+        <div
+          className={`${fr.cx("fr-col-xl", getStateMargin())} ${ButtonContainer}`}
+        >
+          <Button
+            type="submit"
+            iconPosition="right"
+            iconId="fr-icon-search-line"
+            className={`${ButtonStyle}`}
+          >
+            Rechercher
+          </Button>
+        </div>
+      </form>
+
+      <div>
+        <div className={fr.cx("fr-mt-2w")}>
+          {!!enterprises?.length && !loading && (
+            <p className={fr.cx("fr-h5")}>
+              {enterprises.length}
+              {enterprises.length > 1
+                ? " entreprises trouvées"
+                : " entreprise trouvée"}
+            </p>
+          )}
+          {loading && (
+            <div className={fr.cx("fr-grid-row")}>
+              <p className={fr.cx("fr-h5", "fr-mb-0")}>Chargement en cours</p>
+              <div
+                className={`${fr.cx("fr-ml-1w", "fr-mt-1w")} ${SpinnerBlock}`}
+              >
+                <Image priority src={Spinner} alt="Chargement en cours" />
+              </div>
+            </div>
+          )}
+          {searchState === "notFoundSearch" && (
+            <Alert
+              title="Vous ne trouvez pas votre entreprise ?"
+              description={
+                <>
+                  <p>Il peut y avoir plusieurs explications à cela :</p>
+                  <ul>
+                    <li>
+                      Votre entreprise a été enregistrée sous un autre nom ou un
+                      autre code : si vous le pouvez, utilisez son numéro Siret.
+                      Ce dernier doit être présent sur votre bulletin de paie.
+                    </li>
+                    <li>
+                      Votre entreprise a un statut particulier : administration
+                      ou établissements publics, associations, secteur agricole,
+                      La Poste, La Croix Rouge etc. ;
+                    </li>
+                    <li>Votre entreprise n’a pas de convention collective.</li>
+                  </ul>
+                </>
+              }
+              severity="info"
+            />
+          )}
+        </div>
+        {!!enterprises?.length &&
+          !loading &&
+          enterprises?.map((enterprise) => (
+            <Card
+              key={enterprise.label}
+              className={fr.cx("fr-mt-2w")}
+              border
+              enlargeLink
+              linkProps={{
+                href: widgetMode
+                  ? `/widgets/convention-collective/${enterprise.siren}${getQueries()}`
```

Comment:
pourquoi on a pas ça :
```suggestion
                  ? `/widgets/convention-collective/entreprise/${enterprise.siren}${getQueries()}`
```

Reply:
parce qu'il n'y a pas l'étape avec les 2 boutons et l'étape recherche par nom de cc dans le widget

Reply:
On peut quand même avoir entreprise dans l'URL pour plus de clarté non ?

Reply:
ok